### PR TITLE
[Backwards Compatibility][OSD] check advanced_settings, default page, loaded sample date, timeline and filter&query

### PR DESCRIPTION
**Pull request by ananzh**
### Description
This PR adds the following bwc tests:
1) verify sample data work properly for bwc
2) verify timeline visualization work properly for bwc
This PR also simplifies check_filter_and_query bwc test.
It first removes Unique Visitors check because even fix
the time interval, the number of unique visitors number
is random. Then it simplifies this bwc test.

### Partially Resolved:
https://github.com/opensearch-project/opensearch-build/issues/705

Signed-off-by: Anan Zhuang <ananzh@amazon.com>
 
### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by ananzh** [comment]
Screen Shot 2021-11-02 at 9 18 24 AM

**Review by kavilla — DISMISSED** [review]

File: cypress/integration/osd/check_default_page.js [hunk 1]
```diff
@@ -39,6 +39,5 @@ describe('verify default landing page work for bwc', () => {
   
   it('the overview page is set as the default landing pag', () => {
```
Comment (kavilla) [inline cypress/integration/osd/check_default_page.js:40]: can we fix this to say `default landing page`

File: cypress/integration/osd/check_timeline.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+const miscUtils = new MiscUtils(cy);
+  
+describe('verify timeline visualization work properly for bwc', () => {
+    beforeEach(() => {
+        miscUtils.visitPage('app/visualize#');
+    });
+    
+    it('timeline visualizations should be saved and named correctly', () => {
+        cy.get('[data-test-subj="visualizationLandingPage"]')
+          .find('[class="euiFormControlLayout__childrenWrapper"]')
```
Comment (kavilla) [inline cypress/integration/osd/check_timeline.js:42]: NIT: does this read like `verify timeline visualization work properly for bwc timeline visualizations should be saved and named correctly` probably can remove some of the duplication.

File: cypress/integration/osd/check_timeline.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          .type('timeline');        
+        cy.get('[data-test-subj="visListingTitleLink-test-timeline"]').should('have.text', 'test-timeline').click();
+        cy.get('[class="view-line"]').contains('.es(*)');
```
Comment (kavilla) [inline cypress/integration/osd/check_timeline.js:45]: can we get more tests like '.es', '.elasticsearch', and '.opensearch'

**Review by kavilla — COMMENTED** [review]
LGTM just some comments

File: cypress/integration/osd/check_default_page.js (hunk 1 above)
Comment (ananzh) [reply cypress/integration/osd/check_default_page.js:40]: will fix

File: cypress/integration/osd/check_timeline.js (hunk 3 above)
Comment (ananzh) [reply cypress/integration/osd/check_timeline.js:45]: will add

**Review by kavilla — DISMISSED** [review]

File: cypress/integration/osd/check_timeline.js (hunk 3 above)
Comment (kavilla) [reply cypress/integration/osd/check_timeline.js:45]: Thanks for adding.

File: cypress/integration/osd/check_timeline.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+    describe('timeline visualizations should work properly', () => {
+        beforeEach(() => {
+           cy.get('[data-test-subj="visualizationLandingPage"]')
+             .find('[data-test-subj="newItemButton"]')
+             .click();     
+           cy.get('[data-test-subj="visType-timelion"]').click();
+         });
+
+         it('.es(*, kibana1=true should report search error', () => {
```
Comment (tmarkley) [inline cypress/integration/osd/check_timeline.js:56]: Can we close the parenthesis on these descriptions? e.g. `.es(*, kibana1=true) should return search error`

Comment (tmarkley) [reply cypress/integration/osd/check_timeline.js:33]: @ananzh do you want to address this?

**Review by tmarkley — APPROVED** [review]

**Review by kavilla — APPROVED** [review]

**Pull request merged**